api/v1/qdrantclustersnapshot_types.go

```diff
@@ -46,6 +46,9 @@ type QdrantClusterSnapshotStatus struct {
 	// The calculated time (in UTC) this snapshot will be deleted, if so.
 	// +optional
 	RetainUntil *metav1.Time `json:"retainUntil,omitempty"`
+	// Duration specifies how long it took for the snapshot to complete
+	// +optional
+	Duration *metav1.Duration `json:"duration,omitempty"`
```

Comment:
see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#units.

`Duration` for the field name is too ambiguous - how about `CompletionTime`, since we speak about that in the comment above?

Reply:
+1

Reply:
Okay. Then, which one should we chose?

```go
CompletionTime *metav1.Duration `json:"completionTime,omitempty"`
```

```go
CompletionTimeSeconds *int `json:"completionTimeSeconds,omitempty"`
```

Reply:
Renamed `Duration` to `CompletionTime`

Reply:
I would go for `CompletionTimeSeconds` if the operation is never shorter than 1s.

Reply:
We will sent this as string to cluster-api. So, it will be much easier to have this as `metav1.Duration`. It will automatically format into human-readable string.

If we use `CompletionTimeSeconds`, then we have to reformat it in the cluster-api side before we show to the user. Also, it will not look good on additional print column.

Reply:
> It will automatically format into human-readable string.

You should then document what is the schema of the field, i.e. how it should be parsed if somebody need that.

Reply:
Added comment showing sample format.